core/qt-ble.cpp

```diff
@@ -673,6 +673,12 @@ dc_status_t qt_ble_open(void **io, dc_context_t *, const char *devaddr, device_d
 
 			ble->preferredService()->writeDescriptor(d, QByteArray::fromHex("0100"));
 			WAITFOR(ble->descriptorWritten(), 1000);
+			if (!ble->descriptorWritten()) {
+				qDebug() << "Failed to enable notifications for characteristic" << c.uuid();
```

Comment:
Should the error message include that this has happened in the Bluetooth / BLE communication, to make this more obvious to the user?

Reply:
Maybe 'Bluetooth: Failed to enable notifications for characteristic'?

Reply:
I added the "Bluetooth:" prefix.